packages/angular/package.json

```diff
@@ -9,7 +9,8 @@
     "build": "ng build --configuration production",
     "test": "jest",
     "test:watch": "yarn test --watch",
-    "lint": "ng lint"
+    "lint": "yarn typecheck && eslint . --ext .js,.jsx,.ts,.tsx",
```

Comment:
Are `lint` scripts currently used in our pipeline?

Reply:
Yeah it's part of the unit testing which gets run when testing PRs and publishing to @next

Reply:
Great! Makes sense